modules/serverless-minimal-requirements-serving.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+//
+// * /knative-serving/scalability-and-performance-serving.adoc
+
+
+:_mod-docs-content-type: CONCEPT
+[id="serverless-minimal-requirements-serving_{context}"]
+= Minimal requirements of OpenShift Serverless Serving
+
+While the default set-up is suitable for medium-sized workloads, it might be over-sized for smaller set-ups or under-sized for high-workload scenarios.
```

Comment:
```suggestion
While the default setup is suitable for medium-sized workloads, it might be over-sized for smaller setups or under-sized for high-workload scenarios.
```
See ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#s